3차/윤형진_예산.py

```diff
@@ -0,0 +1,13 @@
+def solution(d, budget):
+    answer = 0
+    d.sort()
+    sorted_d = d 
+
+    for i in sorted_d :
```

Comment:
```suggestion
    for i in d :
```

이 부분은 불필요한 과정인 것 같아요